Sources/Plot/API/RSSElements.swift

```diff
@@ -43,9 +43,12 @@ public extension Node where Context: RSSChannelContext {
     /// - parameter date: The date the feed was generated.
     /// - parameter timeZone: The time zone of the given `Date` (default: `.current`).
     static func lastBuildDate(_ date: Date,
-                              timeZone: TimeZone = .current) -> Node {
+                              timeZone: TimeZone = .current,
+                              locale: Locale = Locale(identifier: "en_US_POSIX")) -> Node {
```

Comment:
Could you add this to `RSS.dateFormatter` instead of as arguments, since we do want to use a fixed format?

Reply:
Added it there and actually removed it from `SiteMapElements` as it should not matter since date there is formatted using "numbers" only.